backend/models.js

```diff
@@ -385,9 +391,30 @@ const AgencySchema = new Schema({
   },
 });
 
+const NotesSchema = new Schema({
+  _id: {
+    type: String,
+  },
+  /* Notes text */
+  message: {
+    type: String,
+  },
+  /* Identify events part of the same recurrence */
+  recurringID: {
+    type: String,
+  },
+  timeFromEpoch: {
```

Comment:
We added it so that we can easily compare two notes to see which one belonged to the later event. This helped us when we were doing the all future event delete since it was easier for Mongo to just do a greater than comparison when deciding which notes to delete.